packages/apps/sales/src/components/atoms/SaleOrderCard/SaleOrderCard.tsx

```diff
@@ -0,0 +1,219 @@
+    [SaleOrder?.statusSelect?.Finalized, statusSelect],
+  );
+
+  return (
+    <View style={style}>
+      <ObjectCard
+        style={styles.container}
+        leftContainerFlex={2}
+        iconLeftMargin={10}
+        onPress={onPress}
+        upperTexts={{
+          items: [
+            {
+              customComponent: (
+                <SaleOrderCardTitle
+                  title={sequence}
+                  isIconVisible={orderBeingEdited}
+                />
+              ),
+            },
+            {
+              iconName: 'tag-fill',
+              indicatorText: externalReference,
+              hideIfNull: true,
+            },
+            {
+              iconName: 'person-fill',
+              indicatorText: clientPartnerName,
+              numberOfLines: 2,
+              hideIfNull: true,
+            },
+            {
+              iconName: 'building-fill',
+              indicatorText: companyFullName,
+              hideIfNull: true,
+            },
+            {
+              customComponent: <DateDisplay date={orderDate} size={14} />,
+            },
+          ],
+        }}
+        sideBadges={{
+          items: [
+            {
+              customComponent: (
+                <TextUnit
+                  style={styles.textUnit}
+                  title={I18n.t('Sales_WT')}
+                  value={formatPrice(WTPrice)}
+                  unit={currencySymbol}
+                  defaultColor
+                />
+              ),
+            },
+            {
+              customComponent: (
+                <TextUnit
+                  style={styles.textUnit}
+                  title={I18n.t('Sales_ATI')}
+                  value={formatPrice(ATIPrice)}
+                  unit={currencySymbol}
+                />
+              ),
+            },
+          ],
+        }}
+        lowerBadges={
+          displayBadges && {
+            style: styles.badgesContainer,
+            items: [
+              {
+                displayText: getItemTitle(
+                  SaleOrder?.deliveryState,
+                  deliveryState,
+                ),
+                color: getItemColor(SaleOrder?.deliveryState, deliveryState),
+                style: styles.badge,
+              },
+              {
+                displayText: getItemTitle(
+                  SaleOrder?.invoicingState,
+                  invoicingState,
+                ),
+                color: getItemColor(SaleOrder?.invoicingState, invoicingState),
+                style: styles.badge,
+              },
+            ],
+          }
+        }
+      />
+    </View>
+  );
+};
+
+const getStyles = color =>
+  StyleSheet.create({
+    container: {
+      flex: 1,
+      width: '96%',
+      alignSelf: 'center',
+      marginVertical: 3,
+      borderLeftWidth: 7,
+      borderLeftColor: color,
+    },
+    textUnit: {
+      fontSize: 16,
+      fontWeight: 'bold',
+    },
```

Comment:
This is not needed, use really bold of TextUnit component and fontSize xan be set from component props